utils/src/integ-test/java/uk/gov/moj/sdt/test/utils/DBUnitUtility.java

```diff
@@ -1,635 +1,620 @@
+            connection.prepareStatement("SET CONSTRAINTS ALL DEFERRED").execute();
+
+            final FileInputStream inputStream = new FileInputStream(fileName);
+
+            final StringBuilder stringBuilder = new StringBuilder();
+            final InputStreamReader inputStreamReader = new InputStreamReader(inputStream);
+            final BufferedReader reader = new BufferedReader(inputStreamReader);
+            String line = reader.readLine();
+            while (null != line) {
+                stringBuilder.append(' ');
+                stringBuilder.append(line);
+                line = reader.readLine();
+            }
+            final String sqlString = stringBuilder.toString();
+            final Statement stmt = connection.createStatement();
+
+            // Execute the query
+            stmt.executeUpdate(sqlString);
+
+            // Close the result set, statement and the connection
+            stmt.close();
+
+        } catch (final SQLException e) {
+            LOGGER.error("Failure to drop constraints", e);
+        } catch (final FileNotFoundException e) {
+            e.printStackTrace();
+        } catch (final IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Method to disable all constraints and triggers for specific schema prior
+     * to loading data via DBUNIT for a specific integration test.
+     *
+     * @param schemaName name of the schema in which constraints and triggers are to be
+     *                   disabled
+     */
+    public static void prepareForDbunitLoad(final String schemaName) {
+        try {
+            LOGGER.info("Disabling triggers and constraints for schema = " + schemaName);
+            // dropPrimaryKeys();
+            // Get the connection already setup for DBUNIT.
+            final Connection connection = getConnectionInstance(schemaName).getConnection();
+            // connection.prepareStatement ("SET CONSTRAINTS ALL DEFERRED").execute ();
+
+            // Call the reset stored procedure which should have been loaded into
+            // the database by ANT script.
+            final CallableStatement cs = connection.prepareCall("{call SDT_OWNER.Prepare_For_Dbunit_Load(?)}");
+            cs.setString("p_SchemaName", schemaName);
+            //
+            // Close result set and calleable statement.
+            final ResultSet rs = cs.executeQuery();
+            if (rs != null) {
+                rs.close();
+            }
+            cs.close();
+        } catch (final SQLException e) {
+            LOGGER.error(e.toString());
+        }
+    }
+
+    /**
+     * Method to enable all constraints and triggers for specific schema prior
+     * to loading data via DBUNIT for a specific integration test.
+     *
+     * @param schemaName name of the schema in which constraints and triggers are to be
+     *                   enabled
+     */
+    public static void finishDbunitLoad(final String schemaName) {
```

Comment:
Don't forget to remove this method if it is no longer needed.

Reply:
keepiing these for now, uncommented it now.
will delete DBUnitUtility.java, once we are happy with the changes done and having working application